packages/react-ui-components/package.json

```diff
@@ -50,11 +50,13 @@
 	],
 	"dependencies": {
 		"@kelvininc/ui-components": "0.5.3",
+		"@monaco-editor/react": "^4.4.6",
 		"@rjsf/core": "5.0.0-beta.12",
 		"@rjsf/utils": "5.0.0-beta.12",
 		"@rjsf/validator-ajv8": "5.0.0-beta.12",
 		"classnames": "^2.3.1",
-		"fs-extra": "10.0.0"
+		"fs-extra": "10.0.0",
+		"monaco-editor": "^0.34.1"
```

Comment:
isn't this a dev-dependency since we are using this for its types?

Reply:
you're right, changing it :D